### ISSUE TITLE
Show course rating

### ISSUE THREAD

**Issue by ofiralexi** [TechnionYP5779/team1-bot#79]
Description: As a user I would like to get a rating of a course by other users 
relates to #78 . Please try to figure out if there are important rating features we want to save with that issue ( workload, interesting, etc ? )

**Comment by ShaiZeevi** [comment]
Just so I see we're on the same page. An example query could be:
"please provide with an average difficulty level of 3 or lower" ?

**Comment by ofiralexi** [comment]
Yep that's the spirit. But the one responsible for the other issue needs to provide such behavior.

**Comment by ShaiZeevi** [comment]
I'll leave an appropriate comment on that issue
